WPFWijkagent/WijkagentTests/CategorieTest.cs

```diff
@@ -2,8 +2,6 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
-using WijkagentModels;
```

Comment:
is deze niet noodzakelijk voor de offence en dergelijke?

Reply:
Er werden geen klassen gebruikt door deze use